x-pack/plugins/alerting/server/routes/lib/count_usage_of_predefined_ids.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { UsageCounter } from 'src/plugins/usage_collection/server';
+
+export interface CountUsageOfPredefinedIdsOptions {
+  predefinedId?: string;
+  spaceId?: string;
+  usageCounter?: UsageCounter;
+}
+
+export function countUsageOfPredefinedIds({
+  predefinedId,
+  spaceId,
+  usageCounter,
+}: CountUsageOfPredefinedIdsOptions) {
+  if (predefinedId) {
+    if (usageCounter) {
```

Comment:
I did it this way because of my other PR (adding warnings) where I will be checking for something else inside the `if (predefinedId)` block but outside the `if (usageCounter)` block :)